tests/inspectdb/tests.py

```diff
@@ -267,3 +269,32 @@ def test_introspection_errors(self):
         self.assertIn("# Unable to inspect table 'nonexistent'", output)
         # The error message depends on the backend
         self.assertIn("# The error was:", output)
+
+    def test_include_views(self):
+        """
+        Running inspectdb with --include-views should introspect all database
+        views as well as tables.
+        """
+        with connection.cursor() as cursor:
+            try:
+                cursor.execute(
+                    'CREATE VIEW inspectdb_people_view AS SELECT name '
+                    'from inspectdb_people;')
+            except DatabaseError as e:
+                if 'insufficient privileges' in str(e):
+                    self.fail("The test user has no CREATE VIEW privileges")
+                else:
+                    raise
+        try:
+            out = StringIO()
+            call_command('inspectdb', stdout=out)
+            output = out.getvalue()
+            self.assertNotIn("class InspectdbPeopleView(models.Model):", output)
+
+            out = StringIO()
```

Comment:
You don't have to initialize `out` twice. This line can be removed.

Reply:
Thanks for these suggestions, it's all looking much neater now!